posthog/api/person.py

```diff
@@ -299,6 +299,11 @@ def split(self, request: request.Request, pk=None, **kwargs) -> response.Respons
 
         return response.Response({"success": True}, status=201)
 
+    @action(methods=["POST"], detail=True)
```

Comment:
Missing API docs.

Reply:
isn't that automatically generated?

Reply:
For the endpoint yes, but it contains very little relevant information for users to infer how to actually use it.

See https://posthog.com/docs/api#contributing-to-api-docs for a guide on how to add more information and see other code samples in codebase on how we've documented endpoints (search for posthog.api.documentation imports)